rllib/evaluation/sampler.py

```diff
@@ -994,28 +993,18 @@ def _process_observations(
             # Now that all callbacks are done and users had the chance to add custom
             # metrics based on the last observation in the episode, finish up metrics
             # object and append to `outputs`.
-            atari_metrics: List[RolloutMetrics] = _fetch_atari_metrics(base_env)
             if not episode.is_faulty:
-                if atari_metrics is not None:
-                    for m in atari_metrics:
-                        outputs.append(
-                            m._replace(
-                                custom_metrics=episode.custom_metrics,
-                                hist_data=episode.hist_data,
-                            )
-                        )
```

Comment:
This is not the original concern of the PR, but I believe that we should clear our sampler of such logic that distinguishes between environments. We are not transparent about this and user has to deepdive rllib to understand/modify what is going on here.